streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -899,6 +840,7 @@ private StreamTask convertStandbyToActive(final StandbyTask standbyTask, final S
         return activeTaskCreator.createActiveTaskFromStandby(standbyTask, partitions, mainConsumer);
     }
 
+    // This can also be removed, but what about the test cases?
```

Comment:
If the production code is removed also the test cases are not needed anymore.
If you have comments or questions please add them to the PR instead of inline comments in code. Inline comments are easy to overlook and then we have comments that nobody understands in the codebase.